packages/nhsuk-frontend/src/nhsuk/components/card/_index.scss

```diff
@@ -57,15 +57,11 @@ $card-border-hover-color: $color_nhsuk-grey-3;
   @include nhsuk-responsive-padding(5);
 }
 
-.nhsuk-card__heading,
+.nhsuk-card__heading:has(+ .nhsuk-icon),
```

Comment:
Only browsers that support `:has()` will remove the trailing margin below orphaned headings on primary cards… I think we can live with that, no?